source/release-notes/2.9.1.rst

```diff
@@ -0,0 +1,46 @@
+.. _PMM-2.9.1:
+
+================================================================================
+*Percona Monitoring and Management* 2.9.1
+================================================================================
+
+:Date: August 3, 2020
+:Installation: `Installing Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/2.x/install/index-server.html>`_
+
+*PMM* (`Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/index.html>`_)
+is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL
+performance.
+
+Improvements
+================================================================================
+
+* :jirabug:`PMM-6230`: Home Dashboard: Customizations retained across updates
```

Comment:
it's not about the Home dashboard. you can't edit it. its  about :  You  created a NEW  dashboard for you, set it  as the HOME page and it's now  preserved as HONE dashboard after update